addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15782,32 +15792,32 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36410"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36411"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36412"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36413"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36414"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36415"
-msgid "No info available yet."
+msgid "No information available yet."
```

Comment:
@uNiversaI please revert any of these changes as it just brings pointless work for translators. These string will be changed as da-anda mentioned in due time.